sys/include/net/nanocoap.h

```diff
@@ -973,6 +973,25 @@ static inline ssize_t coap_opt_add_format(coap_pkt_t *pkt, uint16_t format)
  */
 ssize_t coap_opt_add_opaque(coap_pkt_t *pkt, uint16_t optnum, const uint8_t *val, size_t val_len);
 
+/**
+ * @brief   Adds one or multiple Uri-Path options in the form '/path' into pkt
+ *
+ * @note Use this only for null-terminated strings.
+ *
+ * @param[in,out] pkt         Packet being built
+ * @param[in]     path        Resource (sub)path
+ *
+ * @pre     ((pkt != NULL) && (path != NULL))
+ *
+ * @return        number of bytes written to pkt buffer
+ * @return        <0 on error
+ * @return        -ENOSPC if no available options or pkt full
+ */
+static inline ssize_t coap_opt_add_uri_path(coap_pkt_t *pkt, const char *path)
```

Comment:
Please relocate this function below `coap_opt_add_string()`, which it depends on now that it is inline. This relocation also puts the name of the function in alphabetical order, which is helpful with all of the functions in this file.